recipe/meta.yaml

```diff
@@ -12,12 +12,20 @@ source:
 
 build:
   script: python build.py {{ version }} {{ version_build }} {{ driver_version }}
-  number: 1
+  number: 2
   skip: True  # [win]
 
 requirements:
+  build:
+    # default on linux/osx is python2
+    - python >=3.6
+  host:
+    - python >=3.6
+    - tqdm
   run:
```

Comment:
As pointed out in #14 and  #12 ,  conda isn't really needed at runtime. So, it makes sense to drop this section

Reply:
I tried removing it in https://github.com/conda-forge/cudatoolkit-dev-feedstock/pull/11/commits/24b834a65a2d91f8b77e56ead32d83cf1eca2314, but this caused a `CondaMultiError`. I haven't followed the rest of the changes in #14 though.

Reply:
When you get time, could you give 
```yaml
requirements:
  host:
    - python >=3
    - conda
```
a try?